beetsplug/play.py

```diff
@@ -19,17 +19,35 @@
 
 from beets.plugins import BeetsPlugin
 from beets.ui import Subcommand
+from beets.ui.commands import PromptChoice
 from beets import config
 from beets import ui
 from beets import util
 from os.path import relpath
 from tempfile import NamedTemporaryFile
+import subprocess
 
 # Indicate where arguments should be inserted into the command string.
 # If this is missing, they're placed at the end.
 ARGS_MARKER = '$args'
 
 
+def play(command_str, paths, open_args, keep_open=False):
+    """Play items in paths with command_str and optional arguments. If
+    keep_open, return to beets, otherwise exit once command runs.
+    """
+    try:
+        if keep_open:
+            command = command_str.split()
```

Comment:
I'd recommend using `beets.util.shlex_split` instead of `str.split` for more predictable command spitting behavior.